channels/chan_websocket.c

```diff
@@ -156,6 +156,7 @@ struct websocket_pvt {
 #define QUEUE_DRAINED "QUEUE_DRAINED"
 #define DRIVER_STATUS "STATUS"
 #define MEDIA_BUFFERING_COMPLETED "MEDIA_BUFFERING_COMPLETED"
+#define DTMF_PRESSED "DTMF_PRESSED"
```

Comment:
Instead of DTMF_PRESSED how about DTMF_END since the frame that triggers the callback is the AST_FRAME_DTMF_END type.